### DISCUSSION TITLE
日期选择控件：可不可以不日期没有值时，选择时显示指定的日期，而不是当前日期

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#44850] [Ideas]
设置一个属性，打开选择面版时，默认显示指定的日期，而不是当前日期

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/43781

`defaultPickerValue` 在这个场景感觉是有用的？@zbw-zbw @zombieJ @MadCcc

**Reply by zbw-zbw** [reply]
确实，虽然这种场景应该比较少

**Comment by ekuku1984** [comment]
这个需求是用在选开始与结束的场景里， 选了开始日期， 希望结束日期这个控件弹出来选择的时候默认在显示开始日期所在的月份， 一些特殊原因不能使用日期范围， 所以才提了这个想法
